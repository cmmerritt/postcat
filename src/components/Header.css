@font-face {
  font-family: 'Caty';
  src: url ('/assets/caty.ttf') format('ttf')
}

header {
  background-color: wheat;
  color: black;
  display: flex;
  align-items: center;
  justify-content: center;
  font-family: Caty;
  font-size: larger;
}

img {
  height: 100px;
}