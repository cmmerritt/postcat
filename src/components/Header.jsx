import React from 'react';
import styles from './header.css';

const Header = () => {
  return (
    <header>
      <img src="../../public/assets/postcat.png" />
      <h1 className={styles.header}>Postcat</h1>
      <img src="../../public/assets/postcat.png" className={styles.img}/>
    </header>
  );
};

export default Header;
