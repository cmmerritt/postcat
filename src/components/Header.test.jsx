import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Header from './Header.jsx';

describe('Header component', () => {
  afterEach(() => cleanup());
  it('renders Header', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
