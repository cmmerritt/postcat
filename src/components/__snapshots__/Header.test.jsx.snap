// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders Header 1`] = `
<DocumentFragment>
  <header>
    <img
      src="../../public/assets/postcat.png"
    />
    <h1>
      Postcat
    </h1>
    <img
      src="../../public/assets/postcat.png"
    />
  </header>
</DocumentFragment>
`;
